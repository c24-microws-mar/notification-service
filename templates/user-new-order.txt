Your order

<html>
<body style="color: #005ea8;">
<div style="background-color: #005ea8; width:100%; color:#fff; font-size:20px; padding: 15px;">Check24 - MicroserviceWorkshop - MyBeatifullStore</div>

Thank you {%name%} for your order

Your order will be delivered to:
{%name%}
{%address%}

Your order contains the following items:

{%items%}

</body></html>
