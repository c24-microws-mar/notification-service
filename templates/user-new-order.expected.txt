Thank you Lucy Cechtelar for your order

Your order will be delivered to:
Lucy Cechtelar
426 Jordy Lodge
Cartwrightshire, SC 88120-6700

Your order contains the following items:

The Offspring - The Offspring 15.00
Smash - The Offspring 20.00
